sixtyfps_compiler/expression_tree.rs

```diff
@@ -776,6 +776,12 @@ impl Expression {
         self.visit(|e| e.visit_recursive(visitor));
     }
 
+    /// Visit itself and each sub expression recursively
```

Comment:
i really wish there was some way to not have to duplicate everything for mut and non mut